docker-compose-tests.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:11.2-alpine
+    restart: always
+    volumes:
+    - db_data:/var/lib/postgresql-test/data
+  graphql-engine:
+    image: hasura/graphql-engine:v1.0.0-beta.8
+    ports:
+    - 9000:8080
+    depends_on:
+    - postgres
+    restart: always
+    environment:
+      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:@postgres:5432/postgres
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+  hasura-service-tests:
+    build:
+      context: ./database-service
+      dockerfile: ./Dockerfile
+      network: host
+      target: unittest
+    restart: "no" 
+    depends_on:
+      - graphql-engine
+    volumes:
+      - ./database-service:/app
+    entrypoint: pytest --hasura-endpoint http://graphql-engine:8080 -ra --junitxml=test-report.xml
```

Comment:
You define the endpoint as being on port `8080` but you listen on port `9000` in the service definition in `docker-compose-test.yml`. Is this intended ?

Reply:
In that case, hasura serves on `http://graphql-engine:8080`. However, access from localhost is through port `9000`. Have you tried another config on your own laptop?